<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oscDialog</class>
 <widget class="QDialog" name="oscDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSC configuration</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable OSC to control Bitwig Audio or Reaper. To interact with your DAW, you need to install the DrivenByMoss OSC extension in your DAW first and to configure it properly. Please note that the incoming UDP port here must match the outgoing port configured in your DAW and vice versa.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOSC">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>enable OSC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>491</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>491</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Destination Host ('localhost' if same system)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Local Port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLocalport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>9000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Remote Port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRemoteport">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>8000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>295</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>oscDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
